src/algorithms/diamondsquare.jl

```diff
@@ -153,11 +161,11 @@ end
 """
     _square!(mat, alg::DiamondSquare, round::Int, corners::AbstractVector{Tuple{Int,Int}})
 
-    Runs the square step of the `DiamondSquare` algorithm on the square defined
-    by `corners` on the matrix `mat`. The midpoint of each edge of this square is interpolated
-    by computing the mean value of the two corners on the edge and the center of the square, and the
-    displacing it. The displacement is drawn according to `alg.H` and round using `displace`
-
+Runs the square step of the `DiamondSquare` algorithm on the square defined by
```

Comment:
sorry for being nitpicking but could have been nice to have the doc updates in a separate commit :-)

Reply:
yeah sorry - I've done that with two toddlers sitting on my lap, because such is my life at the moment